server/lib/flow-event.js

```diff
@@ -6,38 +6,39 @@ var _ = require('lodash');
 var os = require('os');
 var Promise = require('bluebird');
 
-var DNT_ALLOWED_QUERY_PARAMS = [
+var DNT_ALLOWED_DATA = [
   'context',
   'entrypoint',
   'migration',
   'service',
 ];
-var NO_DNT_ALLOWED_QUERY_PARAMS = DNT_ALLOWED_QUERY_PARAMS.concat([
+var NO_DNT_ALLOWED_DATA = DNT_ALLOWED_DATA.concat([
   'utm_campaign',
   'utm_content',
   'utm_medium',
   'utm_source',
   'utm_term'
 ]);
 var HOSTNAME = os.hostname();
-var MAX_PARAM_LENGTH = 100;
+var MAX_DATA_LENGTH = 100;
 var VERSION = 1;
 
 module.exports = function (event, data, request) {
-  var queryParams = _.pick(request.query, isDNT(request) ?
```

Comment:
I genuinely have no idea why or how I wrote this to process query params first time round. Maybe I implemented the server stuff first then forgot to come back and check it after the client-side code landed? Bad skills, pb.